covid-vaccine-faq-bot/functions/auth.private.js

```diff
@@ -64,12 +33,51 @@ async function getEnvironmentVariable(context, environment, key) {
   return envVars.find(variable => variable.key === key);
 }
 
-async function getEnvironmentVariables(context, environment) {
+async function setEnvironmentVariable(
+  context,
+  environment,
+  key,
+  value,
+  override = true
+) {
   const client = context.getTwilioClient();
-  return await client.serverless
-    .services(environment.serviceSid)
-    .environments(environment.sid)
-    .variables.list();
+  try {
+    const currentVariable = await getEnvironmentVariable(
+      context,
+      environment,
+      key
+    );
+    if (currentVariable) {
+      if (currentVariable.value !== value) {
+        if (override) {
+          console.log(`Updating ${key}...`);
+          await currentVariable.update({ value });
+          return true;
+        } else {
```

Comment:
don't need the else's here (not sure how it passed linter)

Reply:
I used the boilerplate code form vaccine standby